recipe/04-find_nvcc_env.patch

```diff
@@ -28,3 +37,11 @@ diff --git a/warp/build_dll.py b/warp/build_dll.py
 
              with ScopedTimer("build_cuda", active=args.verbose):
                  run_cmd(cuda_cmd)
+
+                 ld_inputs.append(quote(cu_out))
+                 ld_inputs.append(
+-                    f'-L"{cuda_home}/lib64" -lcudart_static -lnvrtc_static -lnvrtc-builtins_static -lnvptxcompiler_static -lpthread -ldl -lrt'
++                    f'-L"{cuda_home}/lib64" -L"{os.environ.get("CUDA_TOOLKIT_INCLUDE_DIR", "")}/lib" -lcudart_static -lnvrtc_static -lnvrtc-builtins_static -lnvptxcompiler_static -lpthread -ldl -lrt'
```

Comment:
Is `CUDA_TOOLKIT_INCLUDE_DIR` still set? IF not, can we remove this?

Reply:
It should be necessary for the linux build, I'll try to remove it

Reply:
I'm not sure, it looks like the `nvcc` build is successfull, but `gcc` cannot find `nvrtc-builtins_static`

```sh
nvcc -O3 --compiler-options -fPIC,-fvisibility=hidden -gencode=arch=compute_52,code=sm_52 -gencode=arch=compute_60,code=sm_60 -gencode=arch=compute_61,code=sm_61 -gencode=arch=compute_70,code=sm_70 -gencode=arch=compute_75,code=sm_75 -gencode=arch=compute_80,code=sm_80 -gencode=arch=compute_86,code=sm_86 -gencode=arch=compute_89,code=sm_89 -gencode=arch=compute_90,code=sm_90 -gencode=arch=compute_90,code=compute_90 -t0 --extended-lambda -DNDEBUG -DWP_ENABLE_CUDA=1 -I"$SRC_DIR/warp/native" -DWP_ENABLE_CUTLASS=1 -I"warp/native/cutlass/include" -I"warp/native/cutlass/tools/util/include" -o "$SRC_DIR/warp/native/warp.cu.o" -c "$SRC_DIR/warp/native/warp.cu"
build_cuda took 135693.07 ms
$BUILD_PREFIX/bin/x86_64-conda-linux-gnu-c++  -shared -Wl,-rpath,'$ORIGIN' -Wl,--no-undefined -Wl,--exclude-libs,ALL -o '$SRC_DIR/warp/bin/warp.so' "$SRC_DIR/warp/native/warp.cpp.o" "$SRC_DIR/warp/native/crt.cpp.o" "$SRC_DIR/warp/native/error.cpp.o" "$SRC_DIR/warp/native/cuda_util.cpp.o" "$SRC_DIR/warp/native/mesh.cpp.o" "$SRC_DIR/warp/native/hashgrid.cpp.o" "$SRC_DIR/warp/native/reduce.cpp.o" "$SRC_DIR/warp/native/runlength_encode.cpp.o" "$SRC_DIR/warp/native/sort.cpp.o" "$SRC_DIR/warp/native/sparse.cpp.o" "$SRC_DIR/warp/native/volume.cpp.o" "$SRC_DIR/warp/native/marching.cpp.o" "$SRC_DIR/warp/native/cutlass_gemm.cpp.o" "$SRC_DIR/warp/native/warp.cu.o" -L"$BUILD_PREFIX/targets/x86_64-linux/lib64" -lcudart_static -lnvrtc_static -lnvrtc-builtins_static -lnvptxcompiler_static -lpthread -ldl -lrt
/home/conda/feedstock_root/build_artifacts/warp-lang_1715266158064/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/12.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lnvrtc_static: No such file or directory
/home/conda/feedstock_root/build_artifacts/warp-lang_1715266158064/_build_env/bin/../lib/gcc/x86_64-conda-linux-gnu/12.3.0/../../../../x86_64-conda-linux-gnu/bin/ld: cannot find -lnvrtc-builtins_static: No such file or directory
collect2: error: ld returned 1 exit status
```

Reply:
No problem, I was just wondering if it was  a leftover, if it is necessary let's leave it!